app/players/dwarves/dwarf_warrior.py

```diff
@@ -0,0 +1,17 @@
+from .dwarf import Dwarf
+
+
+class DwarfWarrior(Dwarf):
+    def __init__(self,
```

Comment:
After the opening bracket, write all arguments on a new line, including the first one; also write the closing bracket on a new line